src/AccessibilityInsights.SharedUx/Controls/TestTabs/AutomatedChecksControl.xaml

```diff
@@ -273,11 +288,6 @@
                                                             <Label Margin="4,0,0,0" Padding="0" Style="{StaticResource lblLink}" 
```

Comment:
I believe we can remove this enclosing Label

Reply:
Removed, as well as the string resource that was only used here